packages/components/.cloud-foundry/manifest-dev.yml

```diff
@@ -1,5 +1,5 @@
 ---
 applications:
-  - name: nui-storybook-dev
+  - name: onyx-storybook-dev
```

Comment:
This will also change the URL of the deployment, so we should also update it in the `apps/docs/src/.vitepress/env.ts` file :)